tests/create_tests.py

```diff
@@ -1,19 +1,86 @@
+import os
 from string import Template
+from typing import List
 from git import Repo
+import requests
 
 
-submodules = list(map(lambda x: x.path, Repo(".").submodules))
+def minimal_tests(datasets: List[str], pr_files: List[str]):
+    """Return the minimal test set for a pull request.
 
-template = Template("""from functions import examine
+    Parameters
+    ----------
+    datasets : List[str]
+        List of datasets in the repository.
+    pr_files : List[str]
+        List of files modified by the pull request.
+    
+    Returns
+    -------
+    [type]
+        [description]
```

Comment:
Please explain in more details what the function returns.

Reply:
I clarified the function return and description in docstring.